activity_calendar/tests/test_models.py

```diff
@@ -698,6 +698,43 @@ def test_get_next_activitymoment_cancelled(self):
             timezone.datetime(2021, 9, 9, 0, 0, 0, tzinfo=timezone.utc)
         )
 
+    def test_get_occurrence_at_nonexistent(self):
+        test_date = timezone.datetime(2020, 8, 14, 19, 0, 0, tzinfo=timezone.utc)
+        occurrence = self.activity.get_occurrence_at(test_date)
+        self.assertIsNone(occurrence)
+
+    def test_get_occurrence_at_recurrent(self):
+        test_date = timezone.datetime(2020, 9, 9, 14, 0, 0, tzinfo=timezone.utc)
+        occurrence = self.activity.get_occurrence_at(test_date)
+        self.assertEqual(occurrence.parent_activity, self.activity)
+        self.assertEqual(occurrence.recurrence_id, test_date)
+
+    def test_get_occurrence_at_recurrent_dst(self):
+        test_date = timezone.datetime(2020, 11, 11, 14, 0, 0, tzinfo=timezone.utc)
+        occurrence = self.activity.get_occurrence_at(test_date)
+        self.assertIsNone(occurrence)
+        # Set time to 15:00 instead of 14:00 due to timeshift
+        test_date = timezone.datetime(2020, 11, 11, 15, 0, 0, tzinfo=timezone.utc)
+        occurrence = self.activity.get_occurrence_at(test_date)
+        self.assertEqual(occurrence.parent_activity, self.activity)
+        self.assertEqual(occurrence.recurrence_id, test_date)
+        # recurrence_id should be the date checked and not the date matching the recurrency pattern (i.e. 14:00)
+
+    def test_get_occurrence_at_db_activitymoment(self):
```

Comment:
Duplicate testcase of the one below it? (if not, they still have the same method name)